src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -32,14 +33,12 @@ const propTypes = {
     /** Whether to show welcome header on a particular page */
     shouldShowWelcomeHeader: PropTypes.bool.isRequired,
 
-    /** Whether to show signIn hero image on a particular page */
-    shouldShowSmallScreen: PropTypes.bool.isRequired,
-
     ...withLocalizePropTypes,
 };
 
 function SignInPageContent(props) {
     const {isSmallScreenWidth} = useWindowDimensions();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
```

Comment:
```suggestion
    const {shouldUseNarrowLayout, isSmallScreenWidth} = useResponsiveLayout();
```

any reason to not use `shouldUseNarrowLayout` instead of this?